index.html

```diff
@@ -11,12 +11,11 @@
     </head>
     <body>
         <header>
-            <div id="header">balatonmegkerulok.hu</div>
+            <div id="header">balatonmegkerulok</div>
         </header>
-
         <nav class="navbar ff_comfortaa position-fixed">
             <div class="menu">
-                <button class="navbar-toggler mx-3" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation" style="background-color: rgb(255, 228, 196, 0.5);">
+                <button class="navbar-toggler mx-3" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation" style="background-color: rgb(238, 157, 38);">
```

Comment:
Nem szokás html inline `style=".."` -t használni. Inkább CSS-be szokás rakni.

Reply:
css-be áthelyezve